percy/client.py

```diff
@@ -28,7 +28,7 @@ def environment(self):
         return self._environment
 
     def create_build(self, **kwargs):
-        repo = kwargs.get('repo') or self.environment.repo
```

Comment:
This code is from 2 years ago and I don't totally understand what `self.environment.repo` is supposed to do here. It looks like it basically falls back to the actual name of this github org and branch, which is not how the other core libraries currently work.

For the token-only process to work there needs to be a check for the repo argument, so rather than check for `kwargs.get('repo')` later on in the code I removed the environment repo pickup here. The tests still pass, so that's good, but it would be helpful to understand the historical context of this line @timhaines @djones